packages/read-model/LICENSE

```diff
@@ -0,0 +1,14 @@
+MIT License
```

Comment:
A new package... should it be automatically MIT? Maybe a question for the Product Owner

Reply:
After a talk with @LukasHeinz, we go for Shopsys license instead.